surefire-grouper/pom.xml

```diff
@@ -32,18 +32,53 @@
   <name>Maven Surefire Test-Grouping Support</name>
   <description>Maven Surefire Test-Grouping Support</description>
 
+  <properties>
+    <javacc.version>7.0.10</javacc.version>
+    <generated.sources.dir>${project.basedir}/src/generated/java</generated.sources.dir>
+  </properties>
+
   <build>
     <plugins>
       <plugin>
-        <groupId>org.codehaus.mojo</groupId>
+        <groupId>org.javacc.plugin</groupId>
         <artifactId>javacc-maven-plugin</artifactId>
-        <version>2.6</version>
+        <version>3.0.3</version>
         <executions>
           <execution>
             <id>javacc</id>
             <goals>
               <goal>javacc</goal>
             </goals>
+            <configuration>
+              <sourceDirectory>${project.basedir}/src/main/javacc</sourceDirectory>
```

Comment:
default value is `${basedir}/src/main/javacc`, so looks the same

Reply:
I like things to be obvious and explicit. Devs not familiar with the workings of this plugin will be able to understand better.

Reply:
@sman-81 
Don;t modify default settings unless it is absolutely necessary.
The old plugin is compliant with IDEs, the default layout, where generates sources appear in `target/generated-sources/javacc`.
The `src/main` files are a subject to git commit them. So I disagree to touch this folder.

Reply:
Nowadays the IDEs can be configured to run the compilation and lokkup `target/generated-sources` which is default path.

Reply:
> The `src/main` files are a subject to git commit them. So I disagree to touch this folder.

@Tibor17 
Did you notice `.gitignore` was updated to prevent this?

Take a look at my comment in the Jira-Ticket.